lib/lighthouse-reporter.js

```diff
@@ -93,7 +93,40 @@ async function writeReport(
     categoryReport[category.id] = Math.round(category.score * 100);
   }
 
-  return { categoryReport, htmlReport };
+  const auditReport = {};
+
+  for (const auditName in lighouseResult.lhr.audits) {
+    if (
+      !Object.prototype.hasOwnProperty.call(
```

Comment:
please, invoke this externally of the if statement to improve readability

Reply:
Same validation is used many times in the codebase already. I'll keep for consistency.